src/SFA.DAS.AANHub.Domain/Interfaces/IAanDataContext.cs

```diff
@@ -12,6 +12,7 @@ public interface IAanDataContext
         DbSet<Employer> Employers { get; }
         DbSet<Partner> Partners { get; }
         DbSet<Admin> Admins { get; }
+        DbSet<StagedApprentice> StagedApprentice { get; }
```

Comment:
Rename this to `StagedApprentices`, to be plural

Reply:
![image](https://user-images.githubusercontent.com/46343852/230028085-6db3f2fb-021c-469e-b08c-117ff116d1fc.png)
Changed to plural but am getting the above error. This is after I merged **CSP-683** into `main `and rebased from `main `(I can see the StagedApprentice.sql file in the `.sln` now).

Reply:
Thanks for your help on this @DevenKShah. Closing this now